aquarius/events/metadata_updater.py

```diff
@@ -101,6 +101,10 @@ def __init__(self, oceandb, other_db_index, web3, config):
             self._quiet_time = 0
 
         self._quiet_time = max(self._quiet_time, default_quiet_time)
+        try:
```

Comment:
this method is duplicated between the two classes, can you please move it to the common parent class?